tools/cd_scripts/e2e_test.sh

```diff
@@ -100,6 +105,13 @@ else
     #install git
     sudo yum -y install git
 
+    # install python3-setuptools tools and python3-pip
+    sudo yum -y install gcc python3-devel python3-setuptools redhat-rpm-config
+    sudo yum -y install python3-pip
+    # Downloading composite object requires integrity checking with CRC32c in gsutil.
+    # it requires to install crcmod.
+    sudo pip3 install --no-cache-dir -U crcmod
```

Comment:
Can't we install crcmod without pip3?

Otherwise you need to follow --required-hashes path to install it.
Ref: https://github.com/GoogleCloudPlatform/gcsfuse/pull/1096

Reply:
Fixed